central/processindicator/datastore/datastore_impl.go

```diff
@@ -219,6 +225,36 @@ func (ds *datastoreImpl) RemoveProcessIndicatorsByPod(ctx context.Context, id st
 	return ds.storage.DeleteByQuery(ctx, q)
 }
 
+// GetProcessIndicatorsRiskView retrieves minimal fields from process indicator for risk evaluation
+func (ds *datastoreImpl) GetProcessIndicatorsRiskView(ctx context.Context, q *v1.Query) ([]*views.ProcessIndicatorRiskView, error) {
+	if ok, err := deploymentExtensionSAC.WriteAllowed(ctx); err != nil {
+		return nil, err
+	} else if !ok {
+		return nil, sac.ErrResourceAccessDenied
+	}
+
+	cloned := q.CloneVT()
+	// Add the select fields of the view to the query.
+	cloned.Selects = []*v1.QuerySelect{
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessID).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ContainerName).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessExecPath).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessContainerStartTime).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessCreationTime).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessName).Proto(),
+		pkgSearch.NewQuerySelect(pkgSearch.ProcessArguments).Proto(),
+	}
```

Comment:
I think we should generate that or use reflection to get field names from a struct.

Reply:
I agree, but I think it is out of scope in the interest of time.  So I wrote https://issues.redhat.com/browse/ROX-31087